utils/analyze_convergence.py

```diff
@@ -143,10 +143,10 @@ def add_args(parser):
         help="Downsample volumes to this box size (pixels). Recommended for boxes > 250-300px",
     )
     group.add_argument(
-        "--cuda",
+        "--device",
```

Comment:
Changed the `cuda` argument to `device` to be consistent with all the other places we're using a similar argument. There is no documentation pages where this needs to be changed.